aqt/archives.py

```diff
@@ -188,7 +191,7 @@ def run(self) -> int:
             return 0
         except CliInputError as e:
             self.logger.error("Command line input error: {}".format(e))
-            exit(1)
+            return 1
```

Comment:
You are right; this should definitely be `return 1` rather than `exit(1)`. However, doing this breaks `tests/test_cli.test_cli_invalid_version()`, which expects an invalid version to raise SystemExit rather than `return 1`. I think that test should be rewritten.